08-rails/06-ERB-and-Layouts.md

```diff
@@ -50,37 +50,40 @@ Both code tags and value tags can be combined to create loops with Ruby code tha
 </ul>
 ```
 
-## Using ERB Tags with Sinatra
-To get the most benefit from the special tags that _ERB_ provides us, we will need a way to provide _ERB_ with variables that can be used inside of those tags. To do this, create instance variables within your Sinatra _routes_, and then reference those instance variables inside of your _views_.
+## Using ERB Tags with Rails
+To get the most benefit from the special tags that _ERB_ provides us, we will need a way to provide _ERB_ with variables that can be used inside of those tags.  To do this we can create variables in our controller.
 
 ```ruby
-# in our Sinatra app
-get "/products" do
-  @all_products = Product.all #returns an array of Product instances
-  erb :products # this tells Sinatra to render views/products.erb
+# in our controller app/controllers/books_controller.rb
+def index
+  @books = ["Once and Future King", "Fellowship of the ring",
+              "Ruby: How to Program", "Web Design with HTML, CSS, JavaScript"]
```

Comment:
It might be nice to have something here about the implicit render of the view with the corresponding name (or include an explicit render here)